x-pack/solutions/chat/plugins/workchat-app/public/application/components/assistant/details/assistant_edit_prompt_modal.tsx

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiForm,
+  EuiFormRow,
+  EuiModal,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiSelect,
+  EuiSpacer,
+  EuiTextArea,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '../../../hooks/use_kibana';
+import { useAgentEdition } from '../../../hooks/use_agent_edition';
+import { assistantLabels } from '../i18n';
+
+const USE_CASES = [
```

Comment:
could we add example use cases prompts and the prompts are different? 

If the user modifies the prompt then it updates. 

Theres also a "custom" option which it changes to when the user modifies the prompt. 

These options are selectable and when another option is selected, should update the prompt

Reply:
Added in https://github.com/elastic/kibana/pull/217984/commits/c7f698850bae6f7c88a27353ba50a642b6c5c49f